### PR TITLE
Feature/mc 11573 list secrets

### PR DESCRIPTION
**Pull request by m-orfanos**
### Fixes [MC-11573 ](https://cloud-ops.atlassian.net/browse/MC-11573 )

Adds Kubernetes secret list documentations

### CODE REVIEW

File: source/includes/kubernetes/_k8_secrets.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/secrets</code>
+
+Retrieve a list of all secrets in a given [environment](#administration-environments).
+
+| Attributes                 | &nbsp;                                                                        |
+| -------------------------- | ----------------------------------------------------------------------------- |
+| `id` <br/>_string_         | The id of the secret                                                          |
+| `apiVersion` <br/>_string_ | The API version used to retrieve the secret                                   |
+| `data` <br/>_string_       | Data contains the secret data.                                                |
```
Comment (Gerudhoh) [inline source/includes/kubernetes/_k8_secrets.md]: This sentence is inconsistent from the rest of the entries. I would suggest it be changed to "The data that the secret contains"

File: source/includes/kubernetes/_k8_secrets.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+| `metadata` <br/>_object_   | The metadata of the secret                                                    |
+| `kind` <br/>_string_       | Kind is a string value representing the REST resource this object represents. |
```
Comment (Gerudhoh) [inline source/includes/kubernetes/_k8_secrets.md]: Could we change this to "**A** string value representing the REST resource **that** this object represents."

File: source/includes/kubernetes_extension/_k8_secrets.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+| `cluster_id` <br/>_string_ | The id of the cluster in which to list the secrets. |
+
+| Attributes                 | &nbsp;                                                                        |
+| -------------------------- | ----------------------------------------------------------------------------- |
+| `id` <br/>_string_         | The id of the secret                                                          |
+| `apiVersion` <br/>_string_ | The API version used to retrieve the secret                                   |
+| `data` <br/>_string_       | Data contains the secret data.                                                |
+| `metadata` <br/>_object_   | The metadata of the secret                                                    |
+| `kind` <br/>_string_       | Kind is a string value representing the REST resource this object represents. |
+| `type` <br/>_string_       | Used to facilitate programmatic handling of secret data.                      |
```
Comment (Gerudhoh) [inline source/includes/kubernetes_extension/_k8_secrets.md]: Same as above :)

File: source/includes/kubernetes/_k8_secrets.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+| `type` <br/>_string_       | Used to facilitate programmatic handling of secret data.                      |
```
Comment (Gerudhoh) [inline source/includes/kubernetes/_k8_secrets.md]: Would this sentence make sense? "A string used to facilitate programmatic handling of a secret's data."
Two reasons I suggest this:
1. Consistency with rest of the rows
2. "secret data" makes us sound like spies and I love it but I this "a secret's data" is a more clear/coder approach

Comment (m-orfanos) [reply source/includes/kubernetes/_k8_secrets.md]: Thank you for the effort, all done 73eb6182afe8c40ee4ffd0395735749a59e8cf4f

**Review by Gerudhoh — APPROVED** [review]

**Pull request merged**